mesonbuild/compilers/compilers.py

```diff
@@ -148,6 +148,12 @@ def is_assembly(fname: 'mesonlib.FileOrString') -> bool:
     suffix = fname.split('.')[-1]
     return suffix in assembler_suffixes
 
+def is_java(fname: 'mesonlib.FileOrString') -> bool:
```

Comment:
No need to quote here, I know you copied from older code that was written when this was required, but we're trying to move away from his.

```suggestion
def is_java(fname: mesonlib.FileOrString) -> bool:
```

Reply:
Thanks for the feedback! Committed.